rskj-core/src/test/java/co/rsk/peg/feeperkb/FeePerKbVoteCaller.java

```diff
@@ -0,0 +1,24 @@
+package co.rsk.peg.feeperkb;
+
+import co.rsk.core.RskAddress;
+
+/**
+ * Created to makes easier the way to get RskAddress of 3 authorized and 1 unauthorized Callers. It
+ * contains hex encoded string associated to each caller and return the RskAddress of each one.
+ */
+public enum FeePerKbVoteCaller {
```

Comment:
Are we expecting this to only be used by `FeePerKbIntegrationTest`? If that is the case, we can just declare them as constants in `FeePerKbIntegrationTest`

Reply:
Hey, @apancorb! Take a look at this thread, this topic was discussed in it:

https://github.com/rsksmart/rskj/pull/2433#discussion_r1638970808